### PR TITLE
Added a python wrapper for libfsntfs_attribute_get_data_size

### PR DESCRIPTION
**Pull request by segevrl**

### CODE REVIEW

File: pyfsntfs/pyfsntfs_attribute.c [hunk 1]
```diff
@@ -70,6 +77,12 @@ PyGetSetDef pyfsntfs_attribute_object_get_set_definitions[] = {
 	  "The name of the attribute.",
 	  NULL },
 
+	{ "attribute_data_size",
```
Comment (joachimmetz) [inline pyfsntfs/pyfsntfs_attribute.c:80]: why not just `data_size` ?

File: pyfsntfs/pyfsntfs_attribute.c [hunk 2]
```diff
@@ -51,7 +51,14 @@ PyMethodDef pyfsntfs_attribute_object_methods[] = {
 	  "get_attribute_name() -> Unicode string or None\n"
 	  "\n"
 	  "Returns the name of the attribute." },
-
+	
+	{ "get_attribute_data_size",
```
Comment (joachimmetz) [inline pyfsntfs/pyfsntfs_attribute.c:55]: why not just `get_data_size` ?

**Comment by joachimmetz** [comment]
@segevrl thanks for the proposed changes, I'll make comparable changes in an upcoming commit, to include CI test coverage as well.

**Comment by joachimmetz** [comment]
You likely want to expose valid_data_size as well, also see https://github.com/libyal/libfsntfs/blob/main/documentation/New%20Technologies%20File%20System%20(NTFS).asciidoc#553-non-resident-mft-attribute

**Comment by joachimmetz** [comment]
Changes in https://github.com/libyal/libfsntfs/commit/9f8777cc236c02ff07c13dcb4334e7c9ac091915

**Comment by segevrl** [comment]
Thank you very much, I really appreciate your help!
I don't think I need the VCN range. I think I can identify the first attribute in an attribute chain based on the size field, without needing the start VCN.

To output the size of a file, should I use the valid_data_size or the data_size? In which cases are they different?

**Comment by joachimmetz** [comment]
> To output the size of a file, should I use the valid_data_size or the data_size? In which cases are they different?

Note that size of the attribute is not necessarily size of the file context, e.g. LZNT1 or WOF compression. These are different sizes. Regarding the purpose of valid_data_size this is mentioned in the documentation.

**Pull request closed without merging**